src/DynamoCoreWpf/Properties/Resources.zh-TW.resx

```diff
@@ -2273,11 +2273,11 @@
   <data name="AddStyleButton" xml:space="preserve">
     <value>加入型式</value>
   </data>
-  <data name="ResetCPythonButtonText" xml:space="preserve">
-    <value>重設 CPython</value>
+  <data name="ResetPythonNet3ButtonText" xml:space="preserve">
+    <value>重設 PythonNet3</value>
   </data>
-  <data name="ResetCPythonButtonToolTip" xml:space="preserve">
-    <value>透過重新載入模組來重設 CPython 環境。</value>
+  <data name="ResetPythonNet3ButtonToolTip" xml:space="preserve">
```

Comment:
I think these changes to other language files should be done by localization PR. @zeusongit Any thoughts?